bnd.bnd

```diff
@@ -0,0 +1,18 @@
+Bundle-Category: sling

```

Comment:
Most of this is already covered in https://github.com/apache/sling-parent/blob/821e20cd664144d973f63f5f4201ab71322fa800/pom.xml#L384 (and should be inherited automatically)

Reply:
Didn't see your comment until after I merged. I will update the POM